src/components/Location.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Location.propTypes = {
+  name: PropTypes.string,
+  image: PropTypes.string,
+  id: PropTypes.number,
+}
+
+export default function Location({ image, name, id }) {
+  return (
+    <LocationCard id={id}>
+      <img src={image} alt="" />
+      <span>{name}</span>
```

Comment:
maybe this could be a heading like `h2` or `h3` but you will probably work that out later

Reply:
I changed it to `<p>`, because it's just a little text under the image. :)